src/GitHub.Exports/ViewModels/IInfoPanel.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace GitHub.ViewModels
+{
+    public interface IInfoPanel
+    {
+        string Message { get; set; }
+        MessageType MessageType { get; set; }
+    }
```

Comment:
Interface probably not needed, but added it anyway to have a place to keep the `MessageType` enum and be explicit about things (maybe unit test it later, too)